topic05-git-advanced/book/10.10.md

```diff
@@ -0,0 +1,7 @@
+# Links
+
+Oh shit - http://ohshitgit.com/
+Interactive git cheatsheet - http://ndpsoftware.com/git-cheatsheet.html
+Gitmoji - https://gitmoji.carloscuesta.me/
+Pro Git (free book) - https://git-scm.com/book/en/v2
+GitHub resources - https://services.github.com/classnotes/
```

Comment:
maybe add the Atlassian git tutorials - they seem to be the most user-friendly I found, especially for those new to git - https://www.atlassian.com/git/tutorials